### PR TITLE
Mocha

### PR DESCRIPTION
**Pull request by liushuyu**
Add Mocha test framework and Travis CI

---
This change is [Reviewable](https://reviewable.io/reviews/aosc-dev/website-site-ng/43)

### CODE REVIEW

**Comment by liushuyu** [comment]
>Why your motherf***** didn't use lambda?!!

Because you know, I am a bracket blind person who can't match brackets very well.

>Why Travis CI failed?

Check it out yourself: https://travis-ci.org/liushuyu/website-site-ng/builds/173436390

**Comment by MingcongBai** [comment]
Wow I thought it was @artoria2e5 ... LOL

**Comment by liushuyu** [comment]
>Wow I thought it was @artoria2e5 ... LOL

More of a FAQ, better understand the known problems

File: run_test.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+redis_pid=$(pidof redis-server)
+if [[ ${redis_pid} -gt 0 ]]; then
```
Comment (Artoria2e5) [inline run_test.sh]: Lint: bashism (unsupported by sh)

Comment (liushuyu) [inline run_test.sh]: Well, see `package.json`, the script will be run by `bash ./run_test.sh`

Comment (Artoria2e5) [inline run_test.sh]: Still unacceptable.

**Comment by Artoria2e5** [comment]
Well we should just go ahead and not stick our heads at such a trivial change...

**Comment by liushuyu** [comment]
@Artoria2e5 If you could help fix this, that would be awesome

**Comment by liushuyu** [comment]
@Artoria2e5 Thanks, the tests will sure to fail until the targets get fixed

**Comment by MingcongBai** [comment]
Target?

**Comment by liushuyu** [comment]
@MingcongBai See Travis CI test log

**Pull request merged**